src/Components/ProvisioningWizard/steps/SourceMissing/DirectProviderLink.js

```diff
@@ -30,6 +30,10 @@ const DirectProviderLink = ({ image }) => {
         '/providers/Microsoft.Compute/images/' +
         uploadStatus.options.image_name;
       break;
+    case GCP_PROVIDER:
+      text = 'Launch with Google Cloud console';
+      url = 'https://console.cloud.google.com/welcome';
```

Comment:
I see image builder didn't find a way to launch the image directly either, but can we do better?
If I'm send to the welcome page as a user, I do not feel very comfortable as I might not know what to do next, maybe we can include similar info to what IB did include https://github.com/RedHatInsights/image-builder-frontend/blob/b73b0ecd564f02c5e738e76fab70c4a4d29606e9/src/Components/ImagesTable/ImageLinkDirect.js#L112

you know two lines "click link below and then search for the image by ...", but giving user at least some guidance :)